plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,8 @@ Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.deploy.test
 Bundle-Vendor: Google Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.deploy
-Require-Bundle: com.google.cloud.tools.eclipse.test.dependencies
+Require-Bundle: com.google.cloud.tools.eclipse.test.dependencies,
+ com.google.cloud.tools.eclipse.appengine.localserver
```

Comment:
Import-Package wouldn't work?

Reply:
No, since we're looking to pull in the extension.  An alternative is to pull this in via a `tycho—platform-configuration` `<extraRequirements>` in  pom.xml.

Reply:
Okay. It would be nice to have a comment in file indicating this since I know I'm going to look at this a year from now and have no idea why we did it this way. E.g.
Comment: these next two are required instead of imported because...